CMakeLists.txt

```diff
@@ -0,0 +1,186 @@
+cmake_minimum_required(VERSION 3.10)
+project(notes LANGUAGES CXX)
+set (PROJECT_VERSION 2.0.0)
+set (CMAKE_CXX_STANDARD 14)
+set (CMAKE_CXX_STANDARD_REQUIRED ON)
+
+set(CMAKE_INCLUDE_CURRENT_DIR ON)
+
+set(CMAKE_AUTOMOC ON)
+set(CMAKE_AUTOUIC ON)
+set(CMAKE_AUTORCC ON)
+
+if(NOT DEFINED INSTALL_EXAMPLESDIR)
+    set(INSTALL_EXAMPLESDIR "examples")
```

Comment:
I'm not sure what this is. Are we using it, at all?

Reply:
I think I copied this from a QMake to CMake guide, but we're indeed not using this.